### PR TITLE
Add URLs checker

### PR DESCRIPTION
**Pull request by ImaduddinAMajid**
Do test for every URL in the repository to avoid merging broken
link into the repository.

Ref: #194

### CODE REVIEW

**Comment by karantan** [comment]
@ImaduddinAMajid how did you test this that it works?

**Comment by ImaduddinAMajid** [comment]
I did it using docker on my local machine and it does find the broken links. However, some links that are pointing to private repos are considered broken link as well. Need to find a solution for this.

**Comment by ImaduddinAMajid** [comment]
Example use:
image

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+sudo: required
+
+services:
+  - docker
+
+script:
+  # Link Checks
+  - |
+    docker run -ti --rm -v $PWD:/mnt:ro dkhamsing/awesome_bot --allow-dupe --allow-redirect --allow 403 --white-list https://intra.niteo.co,https://github.com/niteoweb/operations,https://github.com/niteoweb/minisites,https://github.com/niteoweb/ebn,https://github.com/niteoweb/woocart,https://github.com/niteoweb/support,http://docs.niteo.co,http://books.niteo.co,http://videos.niteo.co,https://github.com/issues/assigned,https://github.com/orgs/niteoweb/teams,https://apps.rackspace.com --skip-save-results `find . -name "*.md"`
```
Comment (dz0ny) [inline .travis.yml]: Replace with `make linkcheck` this will prevent having the config in two files

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+https://github.com/niteoweb/ebn,\
+https://github.com/niteoweb/woocart,\
+https://github.com/niteoweb/support,\
+http://docs.niteo.co,\
+http://books.niteo.co,\
+http://videos.niteo.co,\
+https://github.com/issues/assigned,\
+https://github.com/orgs/niteoweb/teams,\
+https://apps.rackspace.com/%0D%0A \
+--skip-save-results `find . -name "*.md"`
```
Comment (dz0ny) [inline Makefile]: --skip-save-results `find . -name "*.md"` > --skip-save-results `find . -iname "*.md"`

**Pull request merged**